docs/admin/upgrading.rst

```diff
@@ -37,9 +37,17 @@ Basic instructions
    Any modifications should have been written within plugins, so that they are not lost on overwriting.
    If this is not the case, take care to maintain your modifications. 
 
-.. note::
+From 1.9 or 1.10 to 1.11
```

Comment:
I think we need to just have integration tests for upgrades eventually for cases we want to support. With e2e testing should be doable.